### PR TITLE
Add v1.6 Release Announcement

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

Buildah v1.6 Release announcement.

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
@rhatdan , @nalind, @vrothberg PTAL.  I'll merge after I get a LGTM and will then social media this.

File: _posts/2019-01-22-new.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: default
+title: Buildah V1.6 Release Announcement
```
Comment (vrothberg) [inline _posts/2019-01-22-new.md]: s/V/v/

File: _posts/2019-01-22-new.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+categories: [new]
+---
+Buildah V1.6 is here with lots of improvements and enhancements.  Check out the [Release Announcement](https://buildah.io/releases/2019/01/22/Buildah-version-v1.6.html) for all the details.
```
Comment (vrothberg) [inline _posts/2019-01-22-new.md]: Here's also a capital `V` (instead of a small one).

Comment (TomSweeneyRedHat) [reply _posts/2019-01-22-new.md]: Thanks @vrothberg , I'll touch them up.

**Comment by rhatdan** [comment]
LGTM

**Comment by vrothberg** [comment]
> @vrothberg I discovered I'd the same V vs v thing in the other news for 1.4 and 1.5. I've updated those too. Can you take a quick peak at the _.new_.md files here and make sure I didn't scrozzle one? I left the RA itself alone, no changes there.

Looks good, thanks!

**Pull request merged**